config/_default/config.yml

```diff
@@ -38,6 +38,13 @@ module:
         - source: archetypes
           target: archetypes
 
+security:
+  funcs:
+    getenv:
+      - ASSET_BUNDLE_ID
+      - REPO_THEME_PATH
+      - PULUMI_CONVERT_URL
```

Comment:
This already exists in the pulumi-hugo repo -- it was added with https://github.com/pulumi/pulumi-hugo/pull/971.